explorer/src/config.ts

```diff
@@ -7,80 +7,84 @@ export enum Environment {
   PROD,
 }
 
-/**
- * Application configuration for the explorer
- */
-export interface ExplorerConfig {
-  /**
-   * The port to run the server on
-   */
-  port: number
-  /**
-   * Whether dev mode is enabled or not
-   */
-  env: Environment
-  /**
-   * The origin of the client, used for CORS purposes
-   */
-  clientOrigin: string
-  /**
-   * The value of the secret used to sign cookies.
-   * Must be at least 32 characters.
-   *
-   * For production usage, make sure this value is kept secret
-   * and has sufficient entropy
-   *
-   */
-  cookieSecret: string
-  /**
-   * The cookie expiration time in milliseconds
-   */
-  cookieExpirationMs: number
-}
+export class Config {
+  static port(env = process.env): number {
+    return parseInt(env.EXPLORER_SERVER_PORT) || 8080
```

Comment:
How do you feel about setting the base here? `parseInt(env.EXPLORER_SERVER_PORT, 10)`

Reply:
Is that not a lint?

Reply:
I don't know if I'd want that to be a lint, sometimes I might want a different base. e.g. hex `0xA`

Reply:
Hmmm, something like `must always provide base`?

Reply:
@j16r https://github.com/smartcontractkit/chainlink/pull/2837